Dockerfile

```diff
@@ -2,11 +2,16 @@
 FROM python:3.9
 
 # Set working directory.
+WORKDIR fastapi-app
 
 # Expose port 8000 for your FastAPI server.
-
-# Install Python requirements using Pip.
+EXPOSE 8000
 
 # Copy your codebase into the Docker container.
+COPY . /fastapi-app
+
+# Install Python requirements using Pip.
+RUN pip install -r requirements.txt
 
 # Run FastAPI server on the port exposed above.
+CMD ["python", "main.py"]
```

Comment:
This isn't quite the right command to use here. To run a FastAPI server, the `uvicorn` command needs to be invoked. You can refer to the FastAPI documentation to figure out the correct command required here (The FastAPI docs also have a section on Dockerfiles, which may be helpful).